build-test.sh

```diff
@@ -615,6 +660,10 @@ while :; do
         msbuildonunsupportedplatform)
             __msbuildonunsupportedplatform=1
             ;;
+        priority)
```

Comment:
Can you update the `usage()` function?

Ideally, I'd think `priority` would take a number, which would default to zero, but if you decide not to do that, at least make the argument "priority_one", as just "priority" seems ambiguous to me.